agents/random_agent.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+
+from agents.agent import Agent
+
+
+class RandomAgent(Agent):
+    def init(self, seed):
+        pass
```

Comment:
Shouldn't we set seed: `np.random.seed(seed)`

Reply:
Right, I overlooked that!